packages/nimble-components/src/mapping/text/index.ts

```diff
@@ -0,0 +1,21 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { Mapping } from '../base';
+import { template } from '../base/template';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-mapping-text': MappingText;
+    }
+}
+
+/**
+ * Maps data values to text.
+ */
+export class MappingText extends Mapping {}
```

Comment:
On the surface (i.e. from a constrained view of just what is present in this PR), this comes across as odd, in that the `MappingText` offers no API, and is just a concrete version of the API presented by `Mapping`, which has no additional derivatives. My presumption is that ultimately there will be other necessary `Mapping` derivatives that have their own API needs, but what isn't clear is whether or not the `Mapping` APIs (notably `label`) are necessary for those derivatives. I see the `label` being used as the display content, and so I'm wondering if we would still being using that for a separate `Mapping` type.